alpha/ui.h

```diff
@@ -1,6 +1,11 @@
 #ifndef __TS_UI__
 #define __TS_UI__
 
+struct button {
```

Comment:
use more specific naming than `button` for namespace consistency.

e.g. TS_UIButton

Reply:
This is temporary, will need look further into how to incorporate all button functionality into io.h